src/reducers/books.js

```diff
@@ -0,0 +1,21 @@
+import {CREATE_BOOK, REMOVE_BOOK} from "../actions";
+import {Book} from "../logic/book";
+import {BOOK_CATEGORIES} from "../logic/book";
+
+const initialState = {
```

Comment:
See also my comment related to this in reducers/filters.js

Why do we need to nest `books` inside this object? Why didn't we just set our state to be equal to the array itself? As such:

`const initialState = [ Book(Math.random(), 'Avengers Infinity', BOOK_CATEGORIES[0]) ]`

This unnecessary nesting adds up lots of operation in your downstream processes (like I suppose why you need to write lots of functions in your `BooksList` component just to render books based on filter value).